config/ccms/ccms_keys.yml

```diff
@@ -3915,6 +3926,23 @@ global_merits:
     :br100_meaning: Meaning not defined in BR100
     :response_type: boolean
     :user_defined: false
+  CLIENT_CIVIL_PARTNER:
+    :generate_block?: false
+    :value: false
+    :br100_meaning: The Client Is In A Civil Partnership
+    :response_type: boolean
+    :user_defined: false
+  APPLICATION_FROM_APPLY:
```

Comment:
there is no test for the CLIENT_CIVIL_PARTNER attribute

Reply:
I somehow managed to duplicate this key, it is not related to this ticket so I have deleted it. The original CLIENT_CIVIL_PARTNER key has a relevant test (omitted from payload)